app/browse_hotels/browse.html

```diff
@@ -56,46 +57,72 @@
 Use of this source code is governed by an MIT-style license that can be foundin the LICENSE file at http://material.angularjs.org/HEAD/license.
 -->
 
-<script type="text/ng-template" id="browse.tmpl.html"><md-dialog aria-label="Hotel Detail" class="fullScreenDialog">
-<form ng-cloak>
-  <md-toolbar>
-    <div class="md-toolbar-tools">
-      <h2>{{hotelInfo.brand_name}} {{hotelInfo.branch_name}}</h2>
-    </div>
-  </md-toolbar>
+<script type="text/ng-template" id="browse.tmpl.html">
+  <md-dialog
+      aria-label="Hotel Detail" class="fullScreenDialog">
+    <form ng-cloak>
+      <md-toolbar>
+        <div class="md-toolbar-tools">
+          <h2>{{hotelInfo.brand_name}} {{hotelInfo.branch_name}}</h2>
+        </div>
+      </md-toolbar>
 
-  <md-dialog-content>
-    <img ng-show="url" ng-src="{{url}}" alt="Description" />
-    <div ng-show="!url"><h4>No Image Provided</h4></div>
-    <div class="md-dialog-content">
-      <div id="hotelInfo">
-        <h3><md-icon md-font-icon="material-icons">home</md-icon>{{hotelInfo.brand_name}} - {{hotelInfo.branch_name}}</h3>
-        <p><md-icon md-font-icon="material-icons">place</md-icon>{{hotelInfo.address}}</p>
-      </div>
+      <md-dialog-content>
+        <img ng-show="url" ng-src="{{url}}" alt="Description"/>
+        <div ng-show="!url"><h4>No Image Provided</h4></div>
+        <div class="md-dialog-content">
+          <div id="hotelInfo">
+            <h3>
+              <md-icon md-font-icon="material-icons">home</md-icon>
+              {{hotelInfo.brand_name}} - {{hotelInfo.branch_name}}
+            </h3>
+            <p>
+              <md-icon md-font-icon="material-icons">place</md-icon>
+              {{hotelInfo.address}}
+            </p>
+            <p>
+              <md-icon md-font-icon="material-icons">description</md-icon>
+              {{hotelInfo.description}}
+            </p>
+          </div>
 
-      <div flex-gt-xs="">
-        <h4>Start date</h4>
-        <md-datepicker ng-model="startDate" md-placeholder="Enter date"></md-datepicker>
-      </div>
-      <div flex-gt-xs="">
-        <h4>End date</h4>
-        <md-datepicker ng-model="endDate" md-placeholder="Enter date"></md-datepicker>
-      </div>
-    </div>
+          <div>
+            <md-chips ng-model="displayTags" md-removable="false" readonly="true"></md-chips>
```

Comment:
the tag should also be able to display the popularity, see the `reservation` for reference